src/types.ts

```diff
@@ -49,28 +49,33 @@ export interface QueryOpts<TGraphQLVariables = OperationVariables> {
   partialRefetch?: boolean;
 }
 
-export interface GraphqlQueryControls<TGraphQLVariables = OperationVariables> {
+export interface QueryControls<TData = any, TGraphQLVariables = OperationVariables> {

```

Comment:
I changed all usages and added TData generic for better typing but I am not sure it will break something. I don't have good knowledge about all code base. Can you check and confirm changes?